zerver/tests/test_auth_backends.py

```diff
@@ -2060,7 +2060,6 @@ def make_idp_initiated_logout_request(
     def get_account_data_dict(self, email: str, name: str) -> Dict[str, Any]:
         return dict(email=email, name=name)
 
-    @override_settings(SAML_ENABLE_SP_INITIATED_SINGLE_LOGOUT=True)
```

Comment:
These deletion can look mysterious when reviewing this, so just noting that these are deleted because `"sp_initiated_logout_enabled": True,` is added in the test IdP config used for tests in `test_extra_settings.py`